### ISSUE TITLE
Path that contains spaces fails resource compilation

### ISSUE THREAD

**Issue by cyberkm** [fastbuild/fastbuild#64]
Place source code of fastbuild in a folder that contains spaces in its name.
In my case it was c:\Users\User Name\Projects\fastbuild

Run fbuild on the code. the Windows res compilation will fail because command line for res compiler was malformed.

**Comment by ffulin** [comment]
Hello.  Thanks for reporting this.

I've fixed the config file to add the missing quotes around the args to the resource compiler. The fix will be part of the v0.88 source code release. I'll leave this issue open until that is release.

In the interim, you can apply the fix locally by adding the missing quotes around the %1 argument in the main fbuild.bff as follows:
```C++
.ResourceCompiler =
[
	.Compiler					= '$WindowsSDKBasePath$\Bin\x86\RC.exe'
	.CompilerOutputExtension	= '.res'
	.CompilerOptions			= '/nologo /fo"%2" "%1"' // Fix here
]
```

**Comment by ffulin** [comment]
v0.88 has now been release and includes a fix for this issue.
